src/services/api/rentService.ts

```diff
@@ -0,0 +1,12 @@
+import axios from '../../axios/axiosConfig';
+
+const rentBook = (id: string | undefined, days: string) => {
+  return axios
+    .post(`/rent/book/${id}/user?rentPeriod=${days}`)
```

Comment:
What are you doing here if id is undefined, for me it is better not to use undefined